### PR TITLE
Prepare frontend for deployment

### PR DESCRIPTION
**Pull request by gohyongjing**

### CODE REVIEW

File: microservices/front-end/vite.config.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [react(), Svgr()],
-  server: {
-    proxy: {
-      "/api": process.env.AUTH_SERVICE_URL ?? "http://localhost:8080",
+export default defineConfig(({ mode }) => {
+  const env = loadEnv(mode, process.cwd())
+  console.log('env '+ JSON.stringify(env));
```
Comment (wz2k) [inline microservices/front-end/vite.config.ts:8]: I don't think this is needed

**Review by wz2k — APPROVED** [review]
LGTM. Just some comments

File: microservices/front-end/vite.config.ts (hunk 1 above)
Comment (gohyongjing) [reply microservices/front-end/vite.config.ts:8]: I need it to set auth server url for deployment

Comment (wz2k) [reply microservices/front-end/vite.config.ts:8]: the comment is just for the console log

Comment (gohyongjing) [reply microservices/front-end/vite.config.ts]: Fixed in [58b5c38](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g37/pull/39/commits/58b5c38fe55b22e7605daeab6768eb7cea965145)

**Pull request merged**